photon-ml/src/main/scala/com/linkedin/photon/ml/evaluation/PrecisionAtKEvaluator.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linkedin.photon.ml.evaluation
+
+import org.apache.spark.rdd.RDD
+
+import com.linkedin.photon.ml.constants.MathConst
+
+/**
+ * Evaluator for binary classification problems
+ * @param k The cut-off rank based on which the precision is computed (precision @ k)
+ * @param labelAndOffsetAndWeights A [[RDD]] of (id, (label, offset, weight)) tuples
+ * @param documentIds Document ids based on which the labels and scores are grouped to form documents in order to
```

Comment:
Yes.

I chose the term "documentId" because this is how precision@k is introduced in the IR field (https://en.wikipedia.org/wiki/Information_retrieval#Precision_at_K).

But you made a good point, let me better document this.